components/02-elements/table/Table.vue

```diff
@@ -0,0 +1,300 @@
+<template>
+  <table
+    class="table"
+    :class="className"
+    tabindex="0"
+    role="table"
+  >
+    <caption
+      v-if="caption"
+      class="table__caption"
+    >
+      {{ caption }}
+    </caption>
+
+    <thead v-if="headCells">
```

Comment:
I think, heading should be always visible. Maybe if `bodyRows` is missing, we should display some placeholder with information? What do you think?

Reply:
I set both thead and tbody to be required, imo we shouldn't render empty table (without content)